src/main/java/com/arpnetworking/clusteraggregator/models/AggregationMode.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2018 Dropbox
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.arpnetworking.clusteraggregator.models;
+
+/**
+ * The aggregation mode applied to data.
+ *
+ * @author Ville Koskela (vkoskela at dropbox dot com)
+ */
+public enum AggregationMode {
```

Comment:
Lets make this an enum with bit flags and then use EnumSet to check.

Reply:
That seems overkill. We can just expand the enum to provide a true/false for both persistence and reaggregation.